roles/openshift_master/templates/master.yaml.v1.j2

```diff
@@ -102,7 +102,7 @@ masterPublicURL: {{ openshift.master.public_api_url }}
 networkConfig:
   clusterNetworkCIDR: {{ openshift.master.sdn_cluster_network_cidr }}
   hostSubnetLength: {{ openshift.master.sdn_host_subnet_length }}
-{% if openshift.common.use_openshift_sdn %}
+{% if openshift.common.use_openshift_sdn or openshift.common.use_nuage %}
   networkPluginName: {{ openshift.common.sdn_network_plugin_name }}
```

Comment:
I don't see anything setting openshift.common.sdn_network_plugin_name when use_nuage is set.

Are there multiple plugin names that can be used with nuage, or is it just a single one that can be set when use_nuage is set to true?